### PR TITLE
🐞[HOLD] Propose a CKB address format

### PR DESCRIPTION
**Pull request by CipherWang**
This format convention only applies to application level.

### CODE REVIEW

**Comment by janx** [comment]
1. Follow Bitcoin's rule exactly and use the same prefix makes it hard to distinguish a normal bitcoin address and ckb address, which may cause confusion. I think we need more discussions here.
2. It should be `Address` instead of `Account`.

**Comment by CipherWang** [comment]
> 1. Follow Bitcoin's rule exactly and use the same prefix makes it hard to distinguish a normal bitcoin address and ckb address, which may cause confusion. I think we need more discussions here.
The wrapped payload for Bitcoin address is 160 bits, and 256 bits for CKB. If CKB supports other length of cell lock in the future, we could change the prefix hex to distinguish them.

> 2. It should be `Address` instead of `Account`.
np

**Comment by zhangsoledad** [comment]
Bitcoin proposes new checksummed base32 format called `Bech32` and then define Segregated Witness addresses using it. see https://github.com/bitcoin/bips/blob/master/bip-0173.mediawiki, it has many advantages over the old base58, if we want to define one for us, I prefer the new `Bech32`.

**Comment by CipherWang** [comment]
cool, I will reconsider it.

**Comment by CipherWang** [comment]
Change the address format proposal to Bech32 format.

**Comment by ashchan** [comment]
> * Follow Bitcoin's rule exactly and use the same prefix makes it hard to distinguish a normal bitcoin address and ckb address, which may cause confusion. I think we need more discussions here.
> * It should be `Address` instead of `Account`.

My 2 cents on the `Address` change. `Account` would too easily confuse people/ourselves with Ethereum's model.

File: rfcs/0000-address-format/0000-address-format.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+CKB Address Format is an application level [lock script][script-define] display recommendation. The lock script consists of three key parameters, including version, args, and binary_hash. In the consideration of user experience, it is necessary to wrap the raw data structure into a verifiable and extensible format.
+
+## Solution
+
+CKB Address Format follows [Bitcoin base32 address format (BIP-173)][bip173] rules, which add a version prefix to the payload (such as binary_hash and private key), and wrap them in **Bech32** encoding and a [BCH checksum][bch].
+
+A Bech32 string is at most 90 characters long and consists of the **human-readable part**, the **separator**, and the **data part**.
+
+The human-readable part is "ckb" for CKB mainnet, and "cbt" for the testnet. The separator is always "1". The data part is consist of version prefix and payload. The version part means the address format version, and currently fixed to 0x00. The payload part encodes lock script by three data fields, payload = type | binary_ref | args.
```
Comment (janx) [inline rfcs/0000-address-format/0000-address-format.md]: `cbt` should be `ckb` according to our previous discussions.

Comment (janx) [inline rfcs/0000-address-format/0000-address-format.md]: Can we add a section for examples?

Comment (ashchan) [reply rfcs/0000-address-format/0000-address-format.md]: You mean ‘ckt’ or ‘cbt’?

Comment (CipherWang) [reply rfcs/0000-address-format/0000-address-format.md]: ckt, np.

**Comment by doitian** [comment]
I'm sorry but it seems that @github is not going to fix the 500 issue. The team could not open this PR in desktop browser, it is inconvenient to discuss in this PR.  Could you please close this and create a new PR instead?

**Pull request closed without merging**